r5/variables.bal

```diff
@@ -0,0 +1,409 @@
+        'type: COMPOSITE,
+        base: ["Resource"],
+        expression: (),
+        default: (),
+        effectiveLevel: SEARCH_PARAM_CATEGORY_RESOURCE_COMMON,
+        preProcessor: (),
+        postProcessor: ()
+    },
+    "_type": {
+        name: "_type",
+        'type: TOKEN,
+        base: ["Resource"],
+        expression: (),
+        default: (),
+        effectiveLevel: SEARCH_PARAM_CATEGORY_RESOURCE_COMMON,
+        preProcessor: (),
+        postProcessor: ()
+    }
+};
+
+# Search Parameter to control search results
+public final readonly & map<CommonSearchParameterDefinition> CONTROL_SEARCH_PARAMETERS = {
+    "_sort": {
+        name: "_sort",
+        'type: STRING,
+        base: ["Resource"],
+        expression: (),
+        default: (),
+        effectiveLevel: SEARCH_PARAM_CATEGORY_SEARCH_RESULT,
+        preProcessor: (),
+        postProcessor: ()
+    },
+    "_count": {
+        name: "_count",
+        'type: NUMBER,
+        base: ["Resource"],
+        expression: (),
+        default: 10,
+        effectiveLevel: SEARCH_PARAM_CATEGORY_SEARCH_RESULT,
+        preProcessor: paginationSearchParamPreProcessor,
+        postProcessor: paginationSearchParamPostProcessor
+    },
+    "_include": {
+        name: "_include",
+        'type: TOKEN,
+        base: ["Resource"],
+        expression: (),
+        default: (),
+        effectiveLevel: SEARCH_PARAM_CATEGORY_SEARCH_RESULT,
+        preProcessor: (),
+        postProcessor: ()
+    },
+    "_revinclude": {
+        name: "_revinclude",
+        'type: TOKEN,
+        base: ["Resource"],
+        expression: (),
+        default: (),
+        effectiveLevel: SEARCH_PARAM_CATEGORY_SEARCH_RESULT,
+        preProcessor: (),
+        postProcessor: ()
+    },
+    "_summary": {
+        name: "_summary",
+        'type: TOKEN,
+        base: ["Resource"],
+        expression: (),
+        default: (),
+        effectiveLevel: SEARCH_PARAM_CATEGORY_SEARCH_RESULT,
+        preProcessor: (),
+        postProcessor: ()
+    },
+    "_elements": {
+        name: "_elements",
+        'type: STRING,
+        base: ["Resource"],
+        expression: (),
+        default: (),
+        effectiveLevel: SEARCH_PARAM_CATEGORY_SEARCH_RESULT,
+        preProcessor: (),
+        postProcessor: ()
+    },
+    "_contained": {
+        name: "_contained",
+        'type: TOKEN,
+        base: ["Resource"],
+        expression: (),
+        default: (),
+        effectiveLevel: SEARCH_PARAM_CATEGORY_SEARCH_RESULT,
+        preProcessor: (),
+        postProcessor: ()
+    },
+    "_containedType": {
+        name: "_containedType",
+        'type: TOKEN,
+        base: ["Resource"],
+        expression: (),
+        default: (),
+        effectiveLevel: SEARCH_PARAM_CATEGORY_SEARCH_RESULT,
+        preProcessor: (),
+        postProcessor: ()
+    }
+};
+
+# Base operations for all resources.
+public final map<FHIROperationDefinition> & readonly BASE_RESOURCE_OPERATIONS = {
```

Comment:
Missing base [operations](https://hl7.org/fhir/R5/operationslist.html#1.4.0);
1. $add
2. $remove
3. $filter
4. $$current-canonical

Reply:
Let's track this in a seperate issue since we don't support the implementation for the operation as well.